pkg/config/apiserver.go

```diff
@@ -251,7 +251,7 @@ func (a *API) Run() {
 
 // getScopes returns the scopes
 // "/scopes"
-func (a *API) getScopes(req *restful.Request, resp *restful.Response) {
+func (a *API) getScopes(_ *restful.Request, resp *restful.Response) {
```

Comment:
Why is this using _ while the other cases above just removed the parameter name completely?